### PR TITLE
Update the containers page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Update the containers page with the latest copy changes.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/containers](http://0.0.0.0:8001/containers)
- Make sure the page matches the [copy doc](https://docs.google.com/document/d/1OtyfNeA--EJgpgvDzqcbJ8jqa6d9H5H9xyxHNxcKXvc/edit#heading=h.o1s37lxyci13)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/3468

### CODE REVIEW

File: templates/containers/index.html [hunk 1]
```diff
@@ -22,86 +19,88 @@ <h2>Ubuntu is the number one platform for containers</h2>
...
+<div class="p-strip is-bordered">
   <div class="row">
     <div class="col-12">
-      <div class="p-heading-icon">
-        <div class="p-heading-icon__header">
-          <img src="{{ ASSET_SERVER_URL }}24e41fc3-kubernetes_logo.svg" alt="Kubernetes logo" class="p-heading-icon__img" />
-          <h2 class="p-heading-icon__title">The Canonical Distribution of Kubernetes<sup>&reg;<a href="#fn1" id="r1">*</a></sup></h2>
-        </div>
-      </div>
+      {% include "shared/_partner-logos.html" with title="Partnering with industry leaders" json_feed_url="http://partners.ubuntu.com/partners.json?technology__name=Containers" feed_item_limit=10 %}
```
Comment (steverydz) [inline templates/containers/index.html]: Logos don't match the copy doc

**Comment by anasereijo** [comment]
@anthonydillon

**Comment by anasereijo** [comment]
@anthonydillon please also check copydoc updates

**Comment by anasereijo** [comment]
@anthonydillon please also check copydoc updates

**Comment by anasereijo** [comment]
@anthonydillon 

screen-shot-2018-06-29-at-11 54 47
screen-shot-2018-06-29-at-11 50 29

- any reason why we have a card on this section? 
screen shot 2018-06-29 at 11 47 37

**Comment by anthonydillon** [comment]
@anasereijo done

**Comment by anasereijo** [comment]
thanks @anthonydillon 
there's still a couple of things left to do:

- make Lxd logo same size as kubernetes
- fix font sizes (as per my jpg)

**Pull request merged**